sdks/python/apache_beam/io/source_test_utils_test.py

```diff
@@ -47,12 +48,18 @@ def _create_source(self, data):
     for bundle in source.split(float('inf')):
       return bundle.source
 
+  @unittest.skipIf(os.environ.get('RUN_SKIPPED_PY3_TESTS') != '1',
+                   'This test still needs to be fixed on Python 3'
+                   'TODO: BEAM-5627')
```

Comment:
> Using the new env var I was able to verify this, and this is in fact a highly recurring error. Only this particular error has the Jira code.

My point is that when we write `#TODO: BEAM-1234` we mean that we expect that the failure will be fixed (or partially fixed) once BEAM-1234 is resolved. So if the test is failing for a different reason than is captured by BEAM-1234, then we should either use a generic Jira "Finish Python 3 porting for module xxx" or file a new Jira - if you think that underlying problem is worth tracking as an issue of its own and/or can be worked on in parallel. That said, are the Jira's you link in this PR accurate?

Reply:
I believe they have a similar origin, yes. I saw that someone is assigned to this issue, so let's merge this PR so that the assignee can take a look. @tvalentyn PTAL at my updated commit